### ISSUE TITLE
[BUG] WPTestCase's _setUpBeforeClass tries to use WordPress code before WordPress is loaded when loadOnly is true

### ISSUE THREAD

**Issue by Luc45** [lucatume/wp-browser#283]
**Environment**
OS: Linux
PHP version: 7.0.33
Installed Codeception version: 3.0.3
Installed wp-browser version: 2.2.17
WordPress version: 4.9

**Codeception configuration file**  

```yaml
actor: IntegrationTester
modules:
    enabled:
        - WPLoader
    config:
        WPLoader:
            loadOnly: true
```

**Suite configuration file**

```yaml
WPDb:
    dsn: 'mysql:host=mysql;dbname=foo_tests'
    user: root
    password: password
    dump:
        # You might want to comment out dump_structure.sql on your local environment so the suite runs faster.
        - tests/_data/dump_structure.sql
        - tests/_data/dump_content.sql
    populate: true
    cleanup: false
    url: 'https://foo.bar'
    tablePrefix: foo_
    urlReplacement: false
    waitlock: 5
    # this can sometimes be faster for importing if the mysql client is available
    # populator: 'mysql --max_allowed_packet=512M -u $user -h $host -p$password $dbname < $dump'
WPLoader:
    wpRootFolder: /srv/www/default/application/wp
    dbName: foo_tests
    dbHost: mysql
    dbUser: root
    dbPassword: password
    wpDebug: true
    dbCharset: utf8
    dbCollate: ''
    tablePrefix: foo_
    domain: foo.bar
    adminEmail: admin@foo.bar
    title: 'WP Tests'
    phpBinary: php
    language: ''
    configFile: tests-config.php
    multisite: true
    isolatedInstall: true
    loadOnly: false
```

**Describe the bug**
When running integration tests with `WPLoader` with `loadOnly: true`, it enqueues the WordPress to be loaded with this function:

`$this->addAction(Events::SUITE_BEFORE, [$this, '_loadWordpress'], 0);`

However, before the event is fired and WordPress is actually loaded, this code is executed:

```php
class WPTestCase extends \tad\WPBrowser\Compat\Codeception\Unit {
    public static function _setUpBeforeClass()
    {
        global $wpdb;
        $wpdb->suppress_errors = false;
   }
}
```

At this point, `$wpdb` is null, because WordPress is not loaded yet, thus throwing a fatal.

**Comment by Luc45** [comment]
Just for testing purposes, I've changed the event to `SUITE_INIT` with priority 100 and the tests ran.

**Comment by Luc45** [comment]
Related https://github.com/lucatume/wp-browser/commit/0b73352aae4cc4fb43663f902530f8e437b6833a

**Comment by Luc45** [comment]
Last called stack before the fatal:

- Enqueue WordPress to be loaded at priority 0
- A couple more things seems to be enqueued with priority 0 as well. See `\Codeception\Subscriber\BeforeAfterTest::$events`
- `beforeClass` runs before `_loadWordPress`. See \Codeception\Subscriber\BeforeAfterTest::beforeClass

**Comment by lucatume** [comment]
Fixed in `2.2.18`
